lib/cylc/conditional_simplifier.py

```diff
@@ -100,31 +101,40 @@ def nest_by_oper( self, nest_me, oper ):
         else:
             return nest_me
 
-    def clean_expr( self, nested_list, criteria ):
-        """Return a list with entries specified by 'critria' removed"""
+    def clean_expr( self, nested_list, criterion ):
+        """Return a list with entries specified by 'criterion' removed"""
```

Comment:
From memory, the intention was to remove one thing at a time rather than a list but I'm going to have to go back through the code with a tooth comb to remember what its inputs should be.